### PR TITLE
(feat) Added configurable end visit label in patient visit header and…

### PR DESCRIPTION
**Pull request by CynthiaKamau**
… actions

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

- Added configurable end visit button label in patient visit header and actions

## Screenshots

https://www.loom.com/share/1925f01828424832ab6ac779e715507a

### CODE REVIEW

**Review by makombe — APPROVED** [review]
LGTM, thanks @CynthiaKamau.

File: packages/esm-patient-chart-app/src/actions-buttons/stop-visit.component.tsx [hunk 1]
```diff
@@ -23,14 +25,16 @@ const StopVisitOverflowMenuItem: React.FC<StopVisitOverflowMenuItemProps> = ({ p
         <button
           className="cds--overflow-menu-options__btn"
           role="menuitem"
-          title={t('endVisit', 'End Visit')}
+          title={!endVisitLabel ? `${t('endVisit', 'End visit')}` : endVisitLabel}
```
Comment (donaldkibet) [inline packages/esm-patient-chart-app/src/actions-buttons/stop-visit.component.tsx:28]:
```suggestion
          title={endVisitLabel ? endVisitLabel :`${t('endVisit', 'End visit')}`}
```

**Review by donaldkibet — APPROVED** [review]
LGTM Thanks @CynthiaKamau

**Comment by brandones** [comment]
@donaldkibet @CynthiaKamau Why did this happen? Why didn't you just use [Translation Overrides](https://github.com/openmrs/openmrs-esm-core/pull/546)? CC @ibacher

**Comment by donaldkibet** [comment]
@brandones  At the time, we were not aware of the support for Translation Overrides, hence why we didn't use it. We'll definitely leverage.

**Pull request merged**